sql/sqlserver/server.go

```diff
@@ -686,7 +690,15 @@ func (s *Server) processInsertRows(node parser.InsertRows) (rows []sqlwire.Resul
 						tmp := string(vt)
 						vals = append(vals, sqlwire.Datum{StringVal: &tmp})
 					case parser.ValArg:
-						return rows, fmt.Errorf("TODO(pmattis): unsupported node: %T", val)
+						tmp := string(vt)
```

Comment:
Add a `TODO` that filling in the placeholder values should really be done as a walk over the parse tree and not something specific to insert rows processing.